src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -9,61 +9,94 @@ import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 import { assetActions } from '../../data/constants/actionTypes';
-import { clearAssetsStatus, deleteAsset } from '../../data/actions/assets';
+import { clearAssetsStatus, deleteAsset, sortUpdate } from '../../data/actions/assets';
 
 export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayAssetsList: [],
       modalOpen: false,
       statusAlertOpen: false,
       assetToDelete: {},
+      deletedAsset: {},
+      deletedAssetIndex: null,
       elementToFocusOnModalClose: {},
-      statusAlertRef: {},
     };
 
+    this.columns = {
+      display_name: {
+        label: 'Name',
+        key: 'display_name',
+        columnSortable: true,
+      },
+      content_type: {
+        label: 'Type',
+        key: 'content_type',
+        columnSortable: true,
+      },
+      date_added: {
+        label: 'Date Added',
+        key: 'date_added',
+        columnSortable: true,
+      },
+      delete_asset: {
+        label: 'Delete Asset',
+        key: 'delete_asset',
+        columnSortable: false,
+      },
+    };
+
+    this.trashcanRefs = {};
+    this.statusAlertRef = {};
+
     this.onDeleteClick = this.onDeleteClick.bind(this);
     this.deleteAsset = this.deleteAsset.bind(this);
     this.addSupplementalTableElements = this.addSupplementalTableElements.bind(this);
     this.closeModal = this.closeModal.bind(this);
     this.closeStatusAlert = this.closeStatusAlert.bind(this);
     this.renderStatusAlert = this.renderStatusAlert.bind(this);
-
-    this.trashcanRefs = {};
   }
 
-  componentDidMount() {
-    this.addSupplementalTableElements();
-  }
+  // componentDidUpdate -> runs after render;
+  // use it for focus
+  // check if set to null
+
+  onSortClick(columnKey) {
+    const sortedColumn = this.props.assetsParameters.sort;
+    const sortedDirection = this.props.assetsParameters.direction;
 
-  componentDidUpdate(prevProps) {
```

Comment:
Since the component now subscribes to the assetsList slice of the Redux store, it will re-render whenever the Redux state changes anyway, so I figure this method is no longer necessary.